### DISCUSSION TITLE
How can I fetch Resturant Menus using google API (using only the placeID, not the accountID)?

### DISCUSSION THREAD

**Discussion by Prottoy2938** [googleapis/googleapis#876] [Q&A]
I have a list of resturants on google with their `placeID`. 

I'm tryna figure out how to fetch their menus using google API or any other method, but so far, I can't find any.

It is possible to do it via the [google business api][1], but you'd need to be the owner of the business (using the accountID) to be able to use the API (& its menu)

But I think there has to be some other way to fetch any restaurants menu (if they are on their google business profile) just by using their place id. 

Blockquote

How can I do it?

  [1]: https://developers.google.com/my-business/reference/rest/v4/FoodMenus

**Comment by parthea** [comment]
@Prottoy2938,

There are API specific support links where you can get answers about API specific functionality. Please follow the support links on an API documentation page to obtain community based support, file issues or submit feature requests.

For example, for the Google Maps Places API, use the following support link:
https://developers.google.com/maps/documentation/places/web-service/support

If the feature doesn't exist on the API, you can file feature requests here:
https://developers.google.com/maps/documentation/places/web-service/support#issue-tracker

In addition, please use the `Send Feedback` button when you're on a Google API documentation page if you have specific questions or feedback on the documentation as the feedback goes directly to the API development and product teams.

**Comment by mcelikx** [comment]
any update?
